tpm2_util.py

```diff
@@ -0,0 +1,73 @@
+from TPM2.Tpm import *
+
+
+def get_random_bytes(tpm: Tpm, n: int) -> list[int]:
```

Comment:
I've got an error here `TypeError: 'type' object is not subscriptable`.
Maybe it is needed to use `List` from `typing`

Reply:
I can run it without errors. Linux specific issue maybe?

Reply:
Or python version maybe 😄